### PR TITLE
Modify levels 4, 6 and 8

### PR DESCRIPTION
**Pull request by maniackcrudelis**
Related to https://github.com/YunoHost/apps/pull/677

**This PR should be merged after https://github.com/YunoHost/apps/pull/678**

### CODE REVIEW

File: package_check.sh [hunk 1]
```diff
@@ -690,6 +700,24 @@ TEST_RESULTS () {
 		fi
 	fi
 
+	# Evaluate the eighth level
+	# -> High quality package.
+	# This level can't be forced to 1
+	if [ "${level[8}" != "auto" ] && [ "${level[8]}" -ne 0 ]
```
Comment (frju365) [inline package_check.sh]: "${level[8}" ? typo ?

Comment (maniackcrudelis) [reply package_check.sh]: yep

File: package_check.sh [hunk 2]
```diff
@@ -690,6 +700,49 @@ TEST_RESULTS () {
...
 
+	# Evaluate the eighth level
+	# -> High quality package.
+	# The level 8 can be validated only by the official list of app.
+	level[8]=0
+	# Define the level 8 only if we're working on a repository. Otherwise, we can't assert that this is the correct app.
+	if echo "$app_arg" | grep --extended-regexp --quiet "https?:\/\/"
+	then
+		# Get the last version of the app list
+		wget -nv https://raw.githubusercontent.com/YunoHost/apps/master/community.json  -O "$script_dir/list.json"
```
Comment (kay0u) [inline package_check.sh]: This should be apps.json, if I well understood, community.json will no longer exists after that.

Comment (maniackcrudelis) [reply package_check.sh]: Absolutely !
Probably a remaining of my tests, apps.json doesn't exist yet. So I was working on community.json.

**Comment by maniackcrudelis** [comment]
Maybe one more review, to validated latest commits, and we're good.

**Pull request merged**